velox/connectors/Connector.h

```diff
@@ -135,29 +137,35 @@ CommitStrategy stringToCommitStrategy(const std::string& strategy);
 /// to be thread-safe.
 class DataSink {
  public:
+  struct Stats {
+    uint64_t completedBytes{0};
+    uint32_t numWrittenFiles{0};
+    common::SpillStats spillStats;
+    /// Contains metadata for each written table partition.
+    std::vector<std::string> partitions;
```

Comment:
Can we rename to `partitionUpdates` as the original name?

Also it would be consistent with:
https://github.com/prestodb/presto/blob/542beb8d89fa86feb15008611f8900f0f316ef5f/presto-hive/src/main/java/com/facebook/presto/hive/PartitionUpdate.java#L34

/// JSON strings for each written table partition.
std::vector<std::string> partitionUpdates;